src/components/dashboard/components/dictionary/DictionaryDetailCard.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import './styles/dictionary-modal.css';
+
+const DictionaryDetailCard = dictionary => {
+  const {
+    dictionary: { name, created_on, updated_on, public_access, owner, owner_type, active_concepts, description }
+  } = dictionary
+
+  return (<div className="dictionaryDetails">
+      <div className="card">
+        <div className="card-header" id="headerDict">
+          <h1 id="headingDict">{ name }</h1>
+        </div>
+        <div className="card-body">
+          <div className="row">
+            <div className="col-6">
+              <h3 id="genCon">General Details</h3>
+              <p id="para">Public Access: { public_access }</p>
+              <p id="para">Owner: { owner }({ owner_type })</p>
```

Comment:
multiple elements should not have the same `id`, you could use a `class` instead.